services/powchain/src/main/java/tech/pegasys/teku/services/powchain/PowchainService.java

```diff
@@ -53,27 +59,29 @@
   private final Eth1DepositManager eth1DepositManager;
   private final Eth1HeadTracker headTracker;
   private final Eth1ChainIdValidator chainIdValidator;
-  private final Web3j web3j;
+  private final List<Web3j> web3js;
+  private final OkHttpClient okHttpClient;
 
   public PowchainService(final ServiceConfig serviceConfig, final PowchainConfiguration powConfig) {
     checkArgument(powConfig.isEnabled());
 
     AsyncRunner asyncRunner = serviceConfig.createAsyncRunner("powchain");
 
-    this.web3j = createWeb3j(powConfig);
+    this.okHttpClient = createOkHttpClient();
+    this.web3js = createWeb3js(powConfig);
 
     final Eth1Provider eth1Provider =
         new ThrottlingEth1Provider(
             new ErrorTrackingEth1Provider(
-                new Web3jEth1Provider(web3j, asyncRunner),
+                createEth1Provider(asyncRunner, this.web3js),
                 asyncRunner,
                 serviceConfig.getTimeProvider()),
             MAXIMUM_CONCURRENT_ETH1_REQUESTS,
             serviceConfig.getMetricsSystem());
 
     final String depositContract = powConfig.getDepositContract().toHexString();
     DepositContractAccessor depositContractAccessor =
-        DepositContractAccessor.create(eth1Provider, web3j, depositContract);
+        DepositContractAccessor.create(eth1Provider, web3js.get(0), depositContract);
```

Comment:
This is fine for now but we'll have to follow up with some reworking of the `DepositContractAccessor` and `DepositContract` so that we don't have to provide a specific `web3js` instance to it.  Otherwise the key task of actually retrieving deposit events won't support fallback.

The `DepositContract` class was originally generated by web3js which is why it's a bit special but since the deposit contract is now fixed we know we won't have to regenerate that code and can adapt it to fit what we need.